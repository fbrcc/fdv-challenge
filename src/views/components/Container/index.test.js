import React from 'react';
import { Container } from './index'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
      .create(<Container title="Ingrese sus datos"></Container>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });