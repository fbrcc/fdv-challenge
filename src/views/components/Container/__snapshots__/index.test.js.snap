// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bdVaJa dhSTzW"
>
  <h4
    className="sc-bwzfXH hmJzJp"
  >
    Ingrese sus datos
  </h4>
</div>
`;
